docs/product-getting-started-guide/src/main/asciidoc/data-object-driver-create-proc.adoc

```diff
@@ -0,0 +1,84 @@
+[id='data-object-driver-proc']
+= Creating the Driver data object
+
+The following chapter instructs you how to create the Driver data object containing data fields, such as: Name, Age, and Total Points.
```

Comment:
See my comment in the Creating the Violation data object section, about intro statements.

Reply:
done